test/1.4/errors/T_EMETH.dml

```diff
@@ -45,6 +50,28 @@ template v3 is u {
     }
 }
 
+group g4 is t {
+    /// ERROR EMETH
+    method indep() {
+    }
+
+    /// ERROR EMETH
+    independent method int_return() -> (int) {
+        return 0;
+    }
+}
+
+group g5 is t {
+    /// ERROR EMETH
+    independent startup method indep() {
+    }
+
+    /// ERROR EMETH
+    independent startup memoized method indep_ret() -> (int) {
+        return 0;
```

Comment:
Some code paths are not testable: trying to override `startup` with non-`startup` errs earlier because `startup` cannot be `default`.